src/components/pages/ParentHome/Progress.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Banner from '../../common/Banner';
+import ParentSidebar from '../ParentHome/ParentSidebar';
+import ProgressBar from '../../common/ProgressBar';
+import '../../../styles/ParentStyles/index.less';
+import { Layout } from 'antd';
+import 'antd/dist/antd.css';
+import { findByLabelText } from '@testing-library/react';
+import ParentNavbar from './ParentNavbar';
+
+const Progress = () => {
+  return (
+    <Layout style={{ width: '100%' }}>
```

Comment:
I would keep all of the styling in the .less file if possible, just to be consistent. Index.less perhaps? Or a separate file?